core/src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -235,7 +235,7 @@
             input -> {
                 Punishment punishment;
 
-                if (Regex.ZERO_OR_N_DIGITS.matches(input.getPrimaryData())) {
+                if (Regex.DIGITS.matches(input.getPrimaryData())) {
```

Comment:
Here I would personally use [`Ints#tryParse(String string)`](https://guava.dev/releases/snapshot/api/docs/com/google/common/primitives/Ints.html#tryParse(java.lang.String)) from [Guava ](https://guava.dev/) (which is available on bukkit).